cmd/kpod/create_cli.go

```diff
@@ -26,25 +26,6 @@ func getAllLabels(labelFile, inputLabels []string) (map[string]string, error) {
 	return labels, nil
 }
 
-func getAllEnvironmentVariables(envFiles, envInput []string) ([]string, error) {
```

Comment:
Curious as to why you're breaking this up. It doesn't seem necessary to completely refactor it out - can we just pass in a spec generator and not return anything?

Reply:
I am not breaking anything up?  I removed a non useful function, that was throwing errors on 
`--env foo`, commands, If you want me to make a separate function for the spec file generation, I can do that.

Reply:
Alright, fair enough. Would be difficult to test anyways, given that we'd only be modifying the spec generator.

Reply:
@mheon These three lines replace the getAllEnvironmentVariables.

```
-       env, err := getAllEnvironmentVariables(c.StringSlice("env-file"), c.StringSlice("env"))
+       env, err := readKVStrings(c.StringSlice("env-file"), c.StringSlice("env"))
        if err != nil {
                return &createConfig{}, errors.Wrapf(err, "unable to process environment variables")
        }
+       env = append(defaultEnvVariables, env...)
```